openwisp_radius/api/serializers.py

```diff
@@ -430,6 +430,8 @@ class RegisterSerializer(
     def validate_phone_number(self, phone_number):
         org = self.context['view'].organization
         if get_organization_radius_settings(org, 'sms_verification'):
+            if phonenumbers.phonenumberutil.number_type(phone_number) != 1:
+                raise serializers.ValidationError(_('Landline numbers can\'t be used to send SMS.'))
```

Comment:
I'd write that only mobile phone numbers are allowed.

Reply:
Okay, I will make the required changes.